executor/executor_simple.go

```diff
@@ -343,6 +345,39 @@ func (e *SimpleExec) executeCreateUser(s *ast.CreateUserStmt) error {
 	return nil
 }
 
+func (e *SimpleExec) executeDropUser(s *ast.DropUserStmt) error {
+	failedUsers := make([]string, 0, len(s.UserList))
+	for _, user := range s.UserList {
+		userName, host := parseUser(user)
+		exists, err := userExists(e.ctx, userName, host)
+		if err != nil {
+			return errors.Trace(err)
+		}
+		log.Warnf("exists %v if exists %v", exists, s.IfExists)
```

Comment:
this log should be more readable, there is no context, we don't know what's happening, it's more like a debug code.
at least we should know it's a drop user action.